collections/_layouts/responsiveness.html

```diff
@@ -0,0 +1,174 @@
+---
+title: Responsiveness
+subtitle: Subtitle
+layout: layout-docs
+category: layouts
+icon: container
+breadcrumb: General
+---
+
+{% capture is-mobile-example %}
+<div class="sgds-container">
+  <div class="row is-mobile">
+    <div class="col">1</div>
+    <div class="col">1</div>
+    <div class="col">1</div>
+    <div class="col">1</div>
+  </div>
+</div>
+{% endcapture %}
+
+{% capture is-desktop-example %}
+<div class="sgds-container">
+  <div class="row is-desktop">
+    <div class="col">1</div>
+    <div class="col">1</div>
+    <div class="col">1</div>
+    <div class="col">1</div>
+  </div>
+</div>
+{% endcapture %}
+
+{% capture is-device-example %}
+<div class="sgds-container">
+  <div class="row is-mobile">
+    <div class="col is-11-mobile is-7-tablet is-5-desktop is-3-widescreen is-1-fullhd">
+      <code>is-11-mobile</code><br>
+      <code>is-7-tablet</code><br>
+      <code>is-5-desktop</code><br>
+      <code>is-3-widescreen</code><br>
+      <code>is-1-fullhd</code>
+    </div>
+    <div class="col">Col</div>
+  </div>
+</div>
+{% endcapture %}
+
+<h4 class="has-text-primary">
+  Handle different grid layouts for each screensize
+</h4>
+
+<hr class="margin--bottom--lg margin--top--lg" />
+
+<h5 class="has-text-weight-semibold margin--bottom--sm">Different column sizes per breakpoint</h5>
+<p>
+    You can define a column size for each viewport size: <b>mobile, tablet, desktop, widescreen and fullHD</b>.
+</p>
+
+
+<table class="table is-bordered is-size-8">
+  <thead>
+    <tr>
+      <td>Class</td>
```

Comment:
I suggest putting a more verbose instruction as to what values x can take to avoid potential confusion. For eg:
<td>Class (Replace x with a value between 1-12) </td>